File: backend/DataAccess/DaoManual/EUC機能/tm_eurptitemDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             EUC帳票項目マスタ
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tm_eurptitemDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tm_eurptitem";
		public string rptid { get; set; }                         //帳票ID
		public string yosikiid { get; set; }                      //様式ID
		public int yosikieda { get; set; }                        //様式枝番
		public string yosikiitemid { get; set; }                  //様式項目ID
		public string reportitemnm { get; set; }                  //帳票項目名称
		public string csvitemnm { get; set; }                     //CSV項目名称
		public string? sqlcolumn { get; set; }                    //SQLカラム
		public string tablealias { get; set; }                    //テーブル別名
		public Enum並び替え orderkbn { get; set; }                 //並び替え
        public int? orderkbnseq { get; set; }                     //並び替えシーケンス
        public int? orderseq { get; set; }                        //並びシーケンス
		public bool reportoutputflg { get; set; }                 //帳票出力フラグ
		public bool csvoutputflg { get; set; }                    //CSV出力フラグ
		public bool headerflg { get; set; }                       //ヘッダーフラグ
		public bool kaipageflg { get; set; }                      //改ページフラグ
		public Enum出力項目区分 itemkbn { get; set; }               //項目区分
		public string? formatkbn { get; set; }                    //フォーマット区分
		public string? formatkbn2 { get; set; }                   //フォーマット区分2
		public string? formatsyosai { get; set; }                 //フォーマット詳細
		public int? height { get; set; }                          //高
		public int? width { get; set; }                           //幅
		public int? offsetx { get; set; }                         //X軸オフセット
		public int? offsety { get; set; }                         //Y軸オフセット
		public string? blankvalue { get; set; }                   //白紙表示
		public string? mastercd { get; set; }                     //名称マスタコード
		public string? masterpara { get; set; }                   //名称マスタパラメータ
		public string? keyvaluepairsjson { get; set; }            //複数のキー・値・ペアjson
		public string? crosskbn { get; set; }                     //集計区分
		//public string? syukeihoho { get; set; }                   //集計方法
		public string? kbn1 { get; set; }                         //小計区分
		public int? level { get; set; }                           //集計レベル
		public string reguserid { get; set; }                     //登録ユーザーID
		public DateTime regdttm { get; set; }                     //登録日時
		public string upduserid { get; set; }                     //更新ユーザーID
		public DateTime upddttm { get; set; }                     //更新日時
    }
}

File: backend/BusinessService/Db/DaoManual/EUC機能/tm_eurptitemDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             EUC帳票項目マスタ
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tm_eurptitemDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tm_eurptitem"
        Public Property rptid As String                         '帳票ID
        Public Property yosikiid As String                      '様式ID
        Public Property yosikieda As Integer                        '様式枝番
        Public Property yosikiitemid As String                  '様式項目ID
        Public Property reportitemnm As String                  '帳票項目名称
        Public Property csvitemnm As String                     'CSV項目名称
        Public Property sqlcolumn As String                    'SQLカラム
        Public Property tablealias As String                    'テーブル別名
        Public Property orderkbn As Enum並び替え                 '並び替え
        Public Property orderkbnseq As Integer                     '並び替えシーケンス
        Public Property orderseq As Integer                        '並びシーケンス
        Public Property reportoutputflg As Boolean                 '帳票出力フラグ
        Public Property csvoutputflg As Boolean                    'CSV出力フラグ
        Public Property headerflg As Boolean                       'ヘッダーフラグ
        Public Property kaipageflg As Boolean                      '改ページフラグ
        Public Property itemkbn As Enum出力項目区分               '項目区分
        Public Property formatkbn As String                    'フォーマット区分
        Public Property formatkbn2 As String                   'フォーマット区分2
        Public Property formatsyosai As String                 'フォーマット詳細
        Public Property height As Integer                          '高
        Public Property width As Integer                           '幅
        Public Property offsetx As Integer                         'X軸オフセット
        Public Property offsety As Integer                         'Y軸オフセット
        Public Property blankvalue As String                   '白紙表示
        Public Property mastercd As String                     '名称マスタコード
        Public Property masterpara As String                   '名称マスタパラメータ
        Public Property keyvaluepairsjson As String            '複数のキー・値・ペアjson
        Public Property crosskbn As String                     '集計区分
        'public string syukeihoho { get; set; }                   //集計方法
        Public Property kbn1 As String                         '小計区分
        Public Property level As Integer                           '集計レベル
        Public Property reguserid As String                     '登録ユーザーID
        Public Property regdttm As Date                     '登録日時
        Public Property upduserid As String                     '更新ユーザーID
        Public Property upddttm As Date                     '更新日時
    End Class
End Namespace
